docs/makeup-hoist-element/index.js

```diff
@@ -0,0 +1,15 @@
+const Hoist = require('../../packages/makeup-hoist-element');
+// const { hoist } = require('../../packages/makeup-modal/node_modules/makeup-hoist-element/src');
+
+let hoisted = false;
```

Comment:
Would it not be better to have a helper to check if the element is hoisted or not? So that the apps do not have to rely on that? Just a thought
So like `Hoist.isHoisted(elementToHoist)`

Reply:
Are you saying get rid of the boolean and manually check if the element is a direct descendant of document.body? I like that actually